xcodeproj/internal/target.bzl

```diff
@@ -937,6 +945,21 @@ def _process_sdk_frameworks(*, objc, build_settings):
          build_settings.get("OTHER_LDFLAGS", [])),
     )
 
+def _process_sdk_dylibs(*, objc, build_settings):
```

Comment:
Can we roll it into `_process_sdk_frameworks`, and rename that function instead? Maybe `_process_sdk_links`?

Reply:
Also can we make sure they appear in the same order that Bazel puts them in?

Reply:
Good idea. I verified it by building the example app:
```
-stdlib=libc++
-std=gnu++11
-arch
arm64
-Xlinker
-objc_abi_version
-Xlinker
2
-fobjc-link-runtime
-ObjC
-framework
Foundation
-framework
ExampleFramework
-framework
ExternalFramework
-weak_framework
SwiftUI
-lc++
-filelist
bazel-out/ios-sim_arm64-min15.0-applebin_ios-ios_sim_arm64-fastbuild-ST-eeab8ca40dab/bin/Example/Example-linker.objlist
```

Looks like it's correct as it is now. First `-framework`, `-weak_framework` and then `-l`.